### ISSUE TITLE
Error 404 to install fluentd

### ISSUE THREAD

**Issue by Tedezed** [kubernetes/contrib#2183]
Hi, today use a playbook of Kubernetes and I have the next error:

```
 TASK [node : Install fluentd pod into each node] *******************************
fatal: [192.168.33.11]: FAILED! => {"changed": false, "dest": "/etc/kubernetes/manifests", "failed": true, "gid": 0, "group": "root", "mode": "0755", "msg": "Request failed", "owner": "root", "response": "HTTP Error 404: Not Found", "size": 4096, "state": "directory", "status_code": 404, "uid": 0, "url": "https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/saltbase/salt/fluentd-es/fluentd-es.yaml"}
	to retry, use: --limit @/home/administrador/vagrant_machines/kubernetes_test/contrib/ansible/playbooks/deploy-cluster.retry
```
My solution, edit `fluentd-install.yml` route `contrib/ansible/roles/node/tasks/fluentd-install.yml`

Remplace: `https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/saltbase/salt/fluentd-es/fluentd-es.yaml`

Use: `https://raw.githubusercontent.com/kubernetes/kops/master/vendor/k8s.io/kubernetes/cluster/saltbase/salt/fluentd-es/fluentd-es.yaml`

**Comment by dustymabe** [comment]
this looks like duplicate of https://github.com/kubernetes/contrib/issues/2159

**Comment by ingvagabund** [comment]
Resolved with #2174
